src/Modules/StatCan.OrchardCore.Widgets/Views/DigitalInnovationLogo.liquid

```diff
@@ -0,0 +1,14 @@
+<svg data-name="DI Logo (mono)" xmlns="http://www.w3.org/2000/svg" width="40.5" height="62" viewBox="0 0 40.488 62">
```

Comment:
Should this be part of the generic widgets module ? or part of another module for our team's assets ?

Reply:
Yes, I've created a new issue for this.